aotools/functions/ell_zernike.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+import numpy as np
+from zernike import RZern

```

Comment:
I can't find the RZern function that you import and use here, is this something else you have added?

Reply:
I use it in the line 33 `zernike = RZern(self.rmax)`.
However, I've thought that maybe I can use zernike from aotools to avoid the use of an extra library. I'll need some days to modify it.

Reply:
I saw that you used it, I mean't that in the `zernike` module there is no `RZern` function, so I wasn't sure if it was something you added to the zernike module but forgot to include in the pull request.

Definitley use the existing functions in aotools if you can, its just that I couldn't find this one defined in aotools.